router/usecase/candidate_routes_test.go

```diff
@@ -126,20 +126,25 @@ func (s *RouterTestSuite) TestCandidateRouteSearcher_HappyPath() {
 		tokenIn       sdk.Coin
 		tokenOutDenom string
 	}{
+		// {
+		// 	name:          "UOSMO -> USDT",
+		// 	tokenIn:       sdk.NewCoin(UOSMO, one),
+		// 	tokenOutDenom: USDT,
+		// },
+		// {
+		// 	name:          "UMEE -> AKT",
+		// 	tokenIn:       sdk.NewCoin(UMEE, one),
+		// 	tokenOutDenom: AKT,
+		// },
+		// {
+		// 	name:          "ALLBTC -> USDC",
+		// 	tokenIn:       sdk.NewCoin(ALLBTC, one),
+		// 	tokenOutDenom: USDC,
+		// },
```

Comment:
Why are these test cases no longer valid?

Reply:
Thanks for catching it - was an accident. Restored now